tests/test_circuit.py

```diff
@@ -136,3 +136,27 @@ def test_adjoint_duplicate_params() -> None:
         exp_adjoint, tuple(values.values()), torch.ones_like(exp_adjoint)
     )[0]
     assert torch.allclose(grad_ad, grad_adjoint)
+
+
+def test_scale() -> None:
```

Comment:
Maybe expand the tests to more cases than the hardcoded ones.

Reply:
did so